tests/BenchmarkDotNet.Tests/ConfigParserTests.cs

```diff
@@ -370,22 +370,53 @@ public void NetFrameworkMonikerParsedCorrectly(string tfm)
         }
 
         [Theory]
-        [InlineData("net50")]
-        [InlineData("net60")]
-        [InlineData("net70")]
-        public void NetMonikersAreRecognizedAsNetCoreMonikers(string tfm)
+        [InlineData("net50", "net5.0")]
+        [InlineData("net60", "net6.0")]
+        [InlineData("net70", "net7.0")]
+        public void NetMonikersAreRecognizedAsNetCoreMonikers(string input, string expected)
         {
-            var config = ConfigParser.Parse(new[] { "-r", tfm }, new OutputLogger(Output)).config;
+            var config = ConfigParser.Parse(new[] { "-r", input }, new OutputLogger(Output)).config;
 
             Assert.Single(config.GetJobs());
             CsProjCoreToolchain toolchain = config.GetJobs().Single().GetToolchain() as CsProjCoreToolchain;
             Assert.NotNull(toolchain);
-            Assert.Equal(tfm, ((DotNetCliGenerator)toolchain.Generator).TargetFrameworkMoniker);
+            Assert.Equal(expected, ((DotNetCliGenerator)toolchain.Generator).TargetFrameworkMoniker);
+        }
+
+        [Theory]
+        [InlineData("net481", ".NET Framework 4.8.1")]
+        [InlineData("net7.0", ".NET 7.0")]
+        public void NetMonikersHaveNiceNames(string input, string expected)
+        {
+            // CoreRuntime.
+            var config = ConfigParser.Parse(new[] { "-r", input }, new OutputLogger(Output)).config;
+
+            Assert.Single(config.GetJobs());
+            var job = config.GetJobs().Single();
+            Assert.NotNull(job);
+            Assert.Equal(expected, job.Environment.Runtime.Name);
+            Assert.Equal(expected, job.GetToolchain().Name);
+        }
+
+        [Theory]
+        [InlineData("nativeaot6.0", "NativeAOT 6.0", "ILCompiler 6.0.0-*")]
+        [InlineData("nativeaot7.0", "NativeAOT 7.0", "Latest ILCompiler")]
```

Comment:
Is `Latest ILCompiler` the desired toolchain name? Should it be `ILCompiler 7.0.0-*`?